scripts/image-builder/config/rhocp-4.14.toml.template

```diff
@@ -1,7 +1,7 @@
-id = "rhocp-4.13"
-name = "Red Hat OpenShift Container Platform 4.13 for RHEL 9"
+id = "rhocp-4.14"
```

Comment:
Should we change this template to use computed values? (in a separate PR)

Reply:
It would be nice, but it's not straightforward on `main` - working on 4.15, we used rhocp-4.14. I could only change this file because 4.14 is already released. On `release-4.15` we still need to use 4.14 until release. We'd have to consult some external information if the repo should be used.

Reply:
Makes sense, I think you've explained that to me before.